container-engine-lib/lib/backend_impls/docker/docker_manager/docker_manager.go

```diff
@@ -1180,10 +1182,10 @@ func (manager *DockerManager) GetContainersByLabels(ctx context.Context, labels
 	return result, nil
 }
 
-// [FetchImage] uses the local [dockerImage] if it's available.
+// [FetchImageMissing] uses the local [dockerImage] if it's available.
 // If unavailable, will attempt to fetch the latest image.
 // Returns error if local [dockerImage] is unavailable and pulling image fails.
-func (manager *DockerManager) FetchImage(ctx context.Context, dockerImage string) (bool, error) {
+func (manager *DockerManager) FetchImageMissing(ctx context.Context, dockerImage string) (bool, error) {
```

Comment:
maybe change to `FetchImageIfMissing` - think `If` expresses the conditional behavior of the function. Without reading the function spec, `FetchImageMissing` is a bit hard to interpret when reading code.